mne/preprocessing/tests/test_ica.py

```
@@ -58,7 +58,7 @@ def _skip_check_picard(method):
     if method == 'picard':
         try:
             import picard  # noqa
-        except Exception as e:
+        except:
```

always `except Exception` not bare except (unless you really mean to do so)